### ISSUE TITLE
Parallel 'mode' not running in Apply but running in Plan

### ISSUE THREAD

**Issue by jeenx** [transcend-io/terragrunt-atlantis-config#170]
Atlantis version: `0.16.1`
terragrunt-atlantis-config version: `1.5.0`

This is my current `repos.yaml` file
```
- id: /.*/
  apply_requirements: [mergeable]
  workflow: terragrunt
  pre_workflow_hooks:
  - run: terragrunt-atlantis-config generate --autoplan --ignore-parent-terragrunt --parallel --create-workspace --automerge --output atlantis.yaml
workflows:
  terragrunt:
    plan:
      steps:
      - env:
          name: TERRAGRUNT_TFPATH
          command: 'echo "terraform${ATLANTIS_TERRAFORM_VERSION}"'
      - run: if [ -f terragrunt.hcl ]; then terragrunt plan -no-color -out $PLANFILE; fi
    apply:
      steps:
      - run: if [ -f terragrunt.hcl ]; then terragrunt apply -no-color $PLANFILE; fi
 ```

All my plans do run in parallel and I noticed the `atlantis.yaml` file is being generated in the `default` workspace.
The file has parallel plan and apply set to true and looks correct.

When I run `atlantis apply` it runs linearly and doesn't parse the `atlantis.yaml`. The apply is executed linearly and the logs don't show its running parallel.

I noticed that the generated workspaces that are created (due to `--create-workspace`) don't have the atlantis.yaml file in the directory. Could this be why its not running parallel since the generated workspaces don't have the atlantis.yaml file? What do I need to get my applies to run in parallel just like my plans do?

**Comment by dmattia** [comment]
Do you know if this happens in newer Atlantis versions? I remember 0.16.1 had some issues with `pre-workflow-hooks` that were fixed in 0.17.0

**Comment by jeenx** [comment]
@dmattia Apologies for the delayed response, I went ahead and updated to atlantis  to `0.17.3` and terragrunt-atlantis-config to `1.9.0`. Parallel applies are now working with no config changes. Thanks again for the swift resposne @dmattia 
Closing the issue.
